libraries/Microsoft.Bot.Builder/Streaming/StreamingRequestHandler.cs

```diff
@@ -28,13 +29,13 @@ namespace Microsoft.Bot.Builder.Streaming
 {
     public class StreamingRequestHandler : RequestHandler
     {
-        private const string _reconnectPath = "api/reconnect";
-
+        private const string ReconnectPath = "api/reconnect";
         private readonly IBot _bot;
         private readonly ILogger _logger;
         private readonly IStreamingActivityProcessor _activityProcessor;
         private readonly string _userAgent;
         private readonly IDictionary<string, DateTime> _conversations;
+        private readonly MicrosoftAppCredentials _appCredentials;
```

Comment:
This should be AppCredentials, not MicrosoftAppCredentials. That way it supports other authentication methods (i.e. certificates.